README.md

```diff
@@ -0,0 +1,80 @@
+# terraform-aws-ldap-maintenance
+
+A step function to maintain LDAP users via slack.
+
+## Overview
+
+This project deploys a collection of lambda functions, an api endpoint, and a step function that will automate disabling LDAP users via an interactive slack message.
+
+## Architecture
+
+![State Machine Definition](_docs/state_machine_def_0.0.1.png)
+
+## Submodules
+
+- [API Gateway](/modules/api_gateway): Deploys an API endpoint that responds asynchronously to slack events and triggers the Slack Listener lambda function
```

Comment:
The lambda functions are also a submodule

Reply:
It seems either API Gateway should be at the heading 3 (e.g., ### API Gateway) level or lambda functions should also be in the submodules list

Reply:
yeah I can see how this layout is confusing.. updated